erehwon/erehwon/templates/profiles/project.html

```diff
@@ -0,0 +1,17 @@
+{% extends "dashboard-layout.html" %}
+
+{% load static %}
+
+{% block main %}
+<section class="idea">
+    <h1> Add/Edit project </h1>
+    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
+    <form method="POST" class="">
+        {% csrf_token %}
+        {{ new_project_form.as_p }}
+        {{ project_update_form.as_p }}
```

Comment:
these two forms should be separated into their own FORM. They do two different things Adding a New Project or Editing an Existing one.